### PR TITLE
[IOPLT-638] new status messages payloads

### PR DESCRIPTION
**Pull request by CrisTofani**
## Short description
Defines a new services' status message to automatically be handled in app.

## List of changes proposed in this pull request
- definitions.yml
- backend.json

## How to test
The discussion about this changes has took place on the related [RFC](https://pagopa.atlassian.net/wiki/spaces/IP/pages/1151402148/IOPLT-597+Evoluzione+della+gestione+dei+Banner+di+disservizio+per+IO+App).

### CODE REVIEW

File: definitions.yml [hunk 1]
```diff
@@ -290,12 +290,15 @@ definitions:
       - is_alive
       - message
       - sections
+      - statusMessages
       - config
```
Comment (CrisTofani) [inline definitions.yml:294]:
```suggestion
      - config
```

File: definitions.yml [hunk 2]
```diff
@@ -311,6 +313,38 @@ definitions:
...
+  StatusMessages:
+    type: object
+    description: The list of status messages by app navigation routes
+    required:
+      - items
+    properties:
+      items:
+        type: array
+        items:
+          $ref: "#/definitions/SpidIdp"
```
Comment (CrisTofani) [inline definitions.yml:325]:
```suggestion
          $ref: "#/definitions/StatusMessage"
```

File: definitions.yml [hunk 3]
```diff
@@ -289,13 +289,14 @@ definitions:
     required:
       - is_alive
       - message
-      - sections
       - config
```
Comment (CrisTofani) [inline definitions.yml:292]:
```suggestion
      - sections
      - config
```

**Review by Vangaorth — APPROVED** [review]
LGTM

Do not forget to report such changes on io-dev-api-server, after the new spec has been tagged and released

**Pull request merged**